src/Service/KeeperDispatcher.cpp

```diff
@@ -61,7 +61,7 @@ void KeeperDispatcher::requestThreadFakeZk(size_t thread_index)
                         request_for_session.request->getOpNum());
                     request_processor->push(request_for_session);
                 }
-                else if (!request_for_session.isForwardRequest())
+                else if (!request_for_session.isForwardRequest() && request_for_session.request->getOpNum() != Coordination::OpNum::Close)
```

Comment:
Why need ```&& request_for_session.request->getOpNum() != Coordination::OpNum::Close``` ?

Reply:
In case of network partition,  separated leader will print illegal log.

Reply:
From ```sessionCleanerTask``` requests ? It is recommended to write in the comment that these close requests come from sessionCleanerTask.